java/java-cloud-run-hello-world/src/test/java/cloudcode/helloworld/web/HelloWorldControllerTests.java

```diff
@@ -0,0 +1,31 @@
+package cloudcode.helloworld.web;
+
```

Comment:
This is awesome test but I didn't see where you actually invoke it as part of CI/CD pipeline.